lib/phut/vhost.rb

```diff
@@ -0,0 +1,73 @@
+require 'phut/null_logger'
+require 'phut/setting'
+require 'phut/shell_runner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.